modules/common/src/main/scala/io/circe/jackson/enrich/package.scala

```diff
@@ -0,0 +1,20 @@
+package io.circe.jackson
+
+import com.fasterxml.jackson.core.{JsonFactory, JsonParser}
+import com.snowplowanalytics.snowplow.enrich.common.utils.CirceUtils
+import io.circe.{Json, ParsingFailure}
+
+import scala.util.control.NonFatal
+
+package object enrich extends JacksonCompat {
```

Comment:
1. Explanation is missing on why we need this hack.
2. This is not a copy-paste from other lib - we should be safe putting our copyright here
3. `io.circe.jackson.enrich` is slightly confusing. `io.circe.jackson.snowplow` maybe? Not much better, but out of ideas. This one is optional.